src/chattextbox/chatTextBox.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import TextField from "@material-ui/core/TextField";
+import Send from "@material-ui/icons/Send";
+import styles from "./styles";
+import { withStyles } from "@material-ui/core/styles";
+
+class ChatTextBoxComponent extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      chatText: "",
+    };
+  }
+
+  submitMessage = () => {
+    if (this.messageValid(this.state.chatText)) {
+      this.props.submitMessageFn(this.state.chatText);
+      document.getElementById("chattextbox").value = "";
+    }
+  };
+
+  userClickedInput = () => console.log("input clicked!");
+
+  userTyping = (e) =>
+    e.keyCode === 13
+      ? this.submitMessage()
+      : this.setState({ chatText: e.target.value });
+
+  messageValid = (text) => text && text.replace(/\s/g, "").length;
```

Comment:
It might be good to pop in a short comment to explain what this validation is doing

Reply:
This line is to check if the user is not sending an empty message. 
I always forget to add comments. Thank you for being attentive. Now I'll try to develop this habit :)